### ISSUE TITLE
question regarding segfault (lxqt-panel/xpad)

### ISSUE THREAD

**Issue by Rosika2** [lxqt/lxqt#1966]
##### System Information

* Distribution & Version: Linux/Lubuntu 20.04.2 LTS
* Kernel: 5.4.0-70
* Qt Version: 5.12.8
* liblxqt Version: 0.14.1

I´m using Lubuntu 20.04 and until now it happened 3 times that **lxqt-panel** _shut down_ and immediately after that _restarted_. In each case **xpad** ("sticky note application for X") also terminated its process.
So my **guess** is that the weird behaviour of lxqt-panel might have something to do with xpad...

Anyhow, I looked at the logs (using **lnav**) each time and I think it´s best to post them here in case they may help:

```
1.)
                                                                                                                            │
│Feb 10 17:59:12 rosika-10159 kernel: [  217.370577] lxqt-panel[1293]: segfault at 8 ip 00007f4dce24dedc sp 00007ffe9049a730 error 4 in libQt5Widgets.so.5.12.8[7f4dce0af000+3c4000]                                                      │
│Feb 10 17:59:12 rosika-10159 kernel: [  217.370591] Code: c0 4d 85 ed 74 08 4c 89 ef e8 60 d0 e6 ff 48 8b 7b 48 4c 89 6b 50 48 89 43 48 48 85 ff 74 0b f0 83 2f 01 75 05 e8 e4 d0 e6 ff <49> 8b 45 08 48 89 98 c8 02 00 00 48 89 6b 08 48│
│Feb 10 17:59:12 rosika-10159 kernel: [  217.948033] traps: xpad[1538] trap int3 ip:7f7ed35d90b5 sp:7ffc45f50e80 error:0 in libglib-2.0.so.0.6400.6[7f7ed359d000+84000]                                                                   │

2.) 
                                                                                   │

│Mar 26 13:35:05 rosika-10159 kernel: [  307.450650] lxqt-panel[1391]: segfault at 8 ip 00007f2beb1a8edc sp 00007fff4c5c7f70 error 4 in libQt5Widgets.so.5.12.8[7f2beb00a000+3c4000]                                                      │
│Mar 26 13:35:05 rosika-10159 kernel: [  307.450663] Code: c0 4d 85 ed 74 08 4c 89 ef e8 60 d0 e6 ff 48 8b 7b 48 4c 89 6b 50 48 89 43 48 48 85 ff 74 0b f0 83 2f 01 75 05 e8 e4 d0 e6 ff <49> 8b 45 08 48 89 98 c8 02 00 00 48 89 6b 08 48│
│Mar 26 13:35:08 rosika-10159 kernel: [  311.231993] traps: xpad[1622] trap int3 ip:7fdcec67e295 sp:7ffe2f2b0230 error:0 in libglib-2.0.so.0.6400.6[7fdcec642000+84000]                                                                   │
3.) 

│Mar 30 18:47:57 rosika-10159 kernel: [20829.104808] lxqt-panel[1406]: segfault at 8 ip 00007f774c1deedc sp 00007ffe85c0ee30 error 4 in libQt5Widgets.so.5.12.8[7f774c040000+3c4000]                                                      │
│Mar 30 18:47:57 rosika-10159 kernel: [20829.104821] Code: c0 4d 85 ed 74 08 4c 89 ef e8 60 d0 e6 ff 48 8b 7b 48 4c 89 6b 50 48 89 43 48 48 85 ff 74 0b f0 83 2f 01 75 05 e8 e4 d0 e6 ff <49> 8b 45 08 48 89 98 c8 02 00 00 48 89 6b 08 48│
│Mar 30 18:47:59 rosika-10159 kernel: [20830.676874] traps: xpad[1620] trap int3 ip:7f354d213295 sp:7fff44367990 error:0 in libglib-2.0.so.0.6400.6[7f354d1d7000+84000] 
```                                                                 

Basically I´m at a loss as what to make of it.
Any help is welcome.

Thanks a lot in advance.
Mana greetings.
Rosika

**Comment by tsujan** [comment]
0.14.1 is outdated.

That being said, a backtrace would be appreciated (consult your distro doc on how to find a meaningful backtrace).
